validation/standard/standard-bytecodes.toml

```diff
@@ -1,3 +1,23 @@
+["op-contracts/v1.8.0"] # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
+# Updated in this release
+system_config = "0x765ed9c632a7af063d046a6a9bbff7fcd638b3eadedc80340cf77416ad1c8c50"             # version 2.3.0
```

Comment:
The proxy has not yet been updated to point at this implementation, that is due to happen after Holocene activates. Until then, we would expect a validation failure on this branch.